packages/zoe/src/contracts/loan/types.js

```diff
@@ -63,7 +66,7 @@
  *
  *   A function to get the current debt
  *
- * @property {PromiseKit} liquidationPromiseKit
+ * @property {PromiseRecord<PriceQuote>} liquidationPromiseKit
```

Comment:
Where is `PromiseRecord` defined? Does it replace `PromiseKit`? Are we switching this naming convention in general? We should discuss before we do. There was a lot nice about "kit".

Reply:
PromiseRecord is actually the name of the type from `@agoric/promise-kit`. `PromiseKit` is not an actual type :) I made no changes here in what names we use, so I think that is outside the scope of this PR as well.